app/(pages)/content-collection/_component/CollectionCard/CollectionCard.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+import styles from './CollectionCard.module.scss';
+import MediaPreview from './MediaPreview';
+
+interface CollectionCardProps {
+  images: string[];
+}
+
+export default function CollectionCard({ images }: CollectionCardProps) {
+  return (
+    <div className={styles.media_container}>
+      <div className={styles.edit_container}>
+        <div className={styles.title_container}>
+          <div className={styles.title}>
+            <p> Wedding name 1 </p>
```

Comment:
Stuff like "Wedding name 1" is expected to be swapped out. When making components like these, I would recommend passing in this data as JSON into the props so there's a smoother transition between static data during development to dynamic data from cloudinary.